frontend/src/hooks/useWebMedia.ts

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react'
+
+export const audioOptions = { audio: true, video: false }
+export const videoOptions = { audio: true, video: true }
+export const photoOptions = { audio: true, video: false }
+
+export const useWebMediaRecorder = (options: any) => {
+  const [audioURL, setAudioURL] = useState({})
+  const [isRecording, setIsRecording] = useState(false)
+  const [recorder, setRecorder]: any = useState(null)
+
+  useEffect(() => {
+    // Lazily obtain recorder first time we're recording.
+    if (recorder === null) {
```

Comment:
Can we shorten this? Not sure if better, just a suggestion.

```suggestion
    if (!recorder) {
```